public/docs/_examples/forms/dart/lib/hero.dart

```diff
@@ -0,0 +1,13 @@
+library hero_form.hero;
+
+class Hero {
+  num number;
```

Comment:
Make this an `int` me thinks – doubt this is meant to be a floating point number